libraries/chain/include/eosio/chain/fileno.hpp

```diff
@@ -0,0 +1,192 @@
+#pragma once
```

Comment:
This should go in fc.
Please add a comment giving credit and a pointer to where the code came from. If you made any modifications to the source please highlight those with comments.